cmake/templates/libxml2Config.cmake.in

```diff
@@ -0,0 +1,40 @@
+if(HUNTER_ENABLED)
+    include(hunter_internal_error)
+    include(hunter_status_debug)
+else()
+    function(hunter_internal_error)
+        message(FATAL_ERROR ${ARGV})
+    endfunction()
+
+    function(hunter_status_debug)
+        message(${ARGV})
+    endfunction()
+endif()
+
+get_filename_component(_install_dir "${CMAKE_CURRENT_LIST_DIR}/../../../" ABSOLUTE)
+
+add_library("libxml2::libxml2" STATIC IMPORTED)
+set_target_properties(libxml2::libxml2 PROPERTIES
+    INTERFACE_INCLUDE_DIRECTORIES "${_install_dir}/include"
+    @TARGET_INTERFACE_COMPILE_DEFINITIONS@
+)
+
+# Get all libxml2Targets-*.cmake files
+get_filename_component(_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+file(GLOB TARGET_FILES "${_DIR}/libxml2Targets-*.cmake")
+foreach(file ${TARGET_FILES})
+    include(${file})
+endforeach()
+
+set(LIBXML2_INCLUDE_DIRS "${_install_dir}/include/libxml2")
+
+# We only build release library right now
+if(CMAKE_CONFIGURATION_TYPES OR CMAKE_BUILD_TYPE)
+    set(LIBXML2_LIBRARY optimized ${LIBXML2_LIBRARY_RELEASE} debug ${LIBXML2_LIBRARY_RELEASE})
+else()
+    # if there are no configuration types and CMAKE_BUILD_TYPE has no value
+    # then just use the release libraries
+    set(LIBXML2_LIBRARY ${LIBXML2_LIBRARY_RELEASE} )
+endif()
+# FIXME: This probably should be set for both cases
+set(LIBXML2_LIBRARIES optimized ${LIBXML2_LIBRARY_RELEASE} debug ${LIBXML2_LIBRARY_RELEASE})
```

Comment:
We already have imported targets no need for old tricks. I think `set(LIBXML2_LIBRARIES libxml2::libxml2)` will work fine. By the way I noticed that in example `LIBXML2_LIBRARIES` with full paths used. It's not very good since will make package that depends on libxml2 unrelocatable.

Don't know what is the good solution here (well, fix in upstream is a best option). Since this package configure `*.pc` file there is mechanism in Hunter to transform it into imported target `PkgConfig::`. See documentation:
* https://docs.hunter.sh/en/latest/creating-new/create/autotools.html

May be we can use that and for compatibility introduce `PkgConfig::` on Visual Studio too (manually, without using helper modules).